src/sagemaker/workflow/pipeline.py

```diff
@@ -175,6 +228,19 @@ def describe(self) -> Dict[str, Any]:
         """
         return self.sagemaker_session.sagemaker_client.describe_pipeline(PipelineName=self.name)
 
+    def display(self):
+        """Prints out a Graphviz DAG visual for the Pipeline
+
+        Returns:
+            A Graphviz object representing the pipeline, if successful.
+        """
+
+        pipelineGraph = PipelineGraph.from_pipeline(self)
+        adjacencyList = pipelineGraph.build_adjacency_list_with_condition_edges()
+        stepStatuses = {}
+
+        return build_visual_dag(self.name, adjacencyList, stepStatuses)
```

Comment:
To unit test the display, we can mock the build_visual_dag method and assert it is called with specific argument
See example here: https://github.com/aws/sagemaker-python-sdk/blob/5f9d0242b2da712507b0664b41335721369f4417/tests/unit/test_xgboost.py#L204-L239